tests/schema/extensions/test_apollo.py

```diff
@@ -117,17 +110,43 @@ async def person(self, info) -> Person:
                         "startOffset": 0,
                         "duration": 0,
                     },
-                    {
-                        "path": ["person", "name"],
-                        "field_name": "name",
-                        "parentType": "Person",
-                        "returnType": "String!",
-                        "startOffset": 0,
-                        "duration": 0,
-                    },
                 ]
             },
             "validation": {"startOffset": 0, "duration": 0},
             "parsing": {"startOffset": 0, "duration": 0},
         }
     }
+
+
+@freeze_time("20120114 12:00:01")
+def test_should_not_trace_instrospection_queries(mocker):
+    mocker.patch(
+        "strawberry.extensions.tracing.apollo.time.perf_counter_ns", return_value=0
+    )
```

Comment:
can we make a copy of this test that is async? Just to be safe 😊

Reply:
Sure. But you think it is a case we can get into? Async introspection queries? 😆

Reply:
Now I got it. I added the test there.